secure/key-manager/src/tests.rs

```diff
@@ -540,7 +528,10 @@ fn verify_init_and_basic_rotation<T: DiemInterface>(mut node: Node<T>) {
     // Verify correct initialization (on-chain and in storage)
     node.key_manager.compare_storage_to_config().unwrap();
     node.key_manager.compare_info_to_config().unwrap();
-    assert_eq!(node.time.now(), node.key_manager.last_rotation().unwrap());
+    assert_eq!(
+        node.time.now().as_secs(),
```

Comment:
Just wondering, we have `time.now().as_secs()` a lot of places here.  Should we just have a convenience function `now_secs()` to get the seconds directly?  When I worked on my own changes with the time service, I found myself doing the `now().as_secs()` too.